### DISCUSSION TITLE
Error: Unable to find any supported Python versions.

### DISCUSSION THREAD

**Discussion by CodestamTechnologies** [vercel/community#6284] [Bug Report]
### Page to Investigate

https://vercel.com/docs/functions/runtimes/python

### Steps to Reproduce

![Screenshot 2024-03-27 102406](https://github.com/vercel/community/assets/144057952/7e8fafae-b358-40c0-ab82-0f7848486417)
![Screenshot 2024-03-27 102401](https://github.com/vercel/community/assets/144057952/f8ef5ca6-1166-4fe9-b0ea-44ae62066555)
![Screenshot 2024-03-27 102349](https://github.com/vercel/community/assets/144057952/24b5095f-2c40-4c04-8dc4-b14e2a969375)
![Screenshot 2024-03-27 102338](https://github.com/vercel/community/assets/144057952/9f8888cc-2717-4d3e-bbe6-e74384074dd1)

here by all the setup is made according to the docs but the issue is still getting to python version
Locally everything is running fine

what might be the problem ?

**Comment by swarnava** [comment]
Can you make sure your Project Settings is set to Node 18.x? 

<img width="806" alt="image" src="https://github.com/vercel/community/assets/1162991/f98713f7-aa6c-4f29-8773-d0aefabfa1de">
